components/starting/StartingNavList.tsx

```diff
@@ -1,28 +1,42 @@
-import gettingStartedPage from '@components/starting/gettingStartedPage';
-import etcPage from '@components/starting/etcPage';
-import somethingPage from '@components/starting/somethingPage';
-import devfilesPage from '@components/starting/devfilesPage';
+import appDevfilesPage from '@components/starting/appDevfilesPage';
+import appStartingPage from '@components/starting/appStartingPage';
+import techDevfilesPage from '@components/starting/techDevfilesPage';
+import techStartingPage from '@components/starting/techStartingPage';
+import enterpriseDevfilesPage from '@components/starting/enterpriseDevfilesPage';
+import enterpriseStartingPage from '@components/starting/enterpriseStartingPage';
+import registryDevfilesPage from '@components/starting/registryDevfilesPage';
+import registryStartingPage from '@components/starting/registryStartingPage';
 
 function StartingNavList() {
   return [
     [
-      'About devfiles',
+      'Application Developer',
       {
-        'What is a devfile': devfilesPage,
-        'etc.': etcPage,
+        'Why devfiles': appDevfilesPage,
+        'Getting started': appStartingPage,
       },
     ],
     [
-      'Develop with devfiles',
+      'Technology and Tools Builders',
       {
-        'Getting started': gettingStartedPage,
-        'Or something': somethingPage,
-        'etc.': etcPage,
+        'Why devfiles': techDevfilesPage,
+        'Getting started': techStartingPage,
+      },
+    ],
+    [
+      'Enterprise Architect and Runtime Provider',
```

Comment:
This header is indented when I test it out for some reason.

Reply:
I opened an issue about this in devfile/api#588 and this should be fixed in a future update, but for now there doesn't seem to be any way around it unfortunately.